ember/app/components/base-barogram.js

```diff
@@ -18,7 +18,7 @@ export default Ember.Component.extend({
   contests: null,
   elevations: [],
 
-  flotStyle: safeComputed('height', height => Ember.String.htmlSafe(`width: 100%; height: ${height}px;`)),
+  flotStyle: conditional('height', htmlSafe(tag`width: 100%; height: ${'height'}px;`)),
```

Comment:
`safeComputed` reads all dependent keys, if one or more are `isNone` it returns `undefined, otherwise it passes them to the callback function. so in this case `undefined` is exactly what I want :+1: